.changelogs/8041.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "\"Fixed support for row and columns headers in parseTable utility.\"",
```

Comment:
The additional quotation marks are not needed here, I guess

Reply:
That's something that we suggest in our changelog generator - but right, additional quotation marks are unnecessary 😄

![image](https://user-images.githubusercontent.com/11849621/124158774-9d12b600-da9a-11eb-98ee-eb68991dc7f2.png)

Reply:
True, true